src/lib/components/SiteFooter/index.jsx

```diff
@@ -358,8 +358,8 @@ class SiteFooter extends React.Component {
               <br />
               <FormattedMessage
                 id="footer.site-footer.trademark-text"
-                defaultMessage="EdX, Open edX, and MicroMasters are registered trademarks of edX Inc. | {icpLicense}"
-                values={{ icpLicense: '粤ICP备17044299号-2' }}
+                defaultMessage="EdX, Open edX, and MicroMasters are registered trademarks of edX Inc. | 深圳市恒宇博科技有限公司 {icpLicense}"
```

Comment:
As a note, I believe edX is always a lowercase e now, even at the start of a sentence.